### PR TITLE
Add command icon

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

Add a command (command key on the keyboard) icon

### Why is it needed?

To be usable for the CMS 

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Comment by alexandrebodin** [comment]
I experimented with some chunking config I don't count on leaving it in the PR but have a look at the analyze and let me know what you think 👍

File: packages/strapi-design-system/webpack.config.js [hunk 1]
```diff
@@ -52,6 +53,14 @@ module.exports = {
     library: 'strapiDs',
     umdNamedDefine: true,
   },
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+      maxInitialRequests: 30,
+      maxAsyncRequests: 30,
+      minSize: 20,
```
Comment (HichamELBSI) [inline packages/strapi-design-system/webpack.config.js:61]: Curious to know why 20bytes to generate a chunk?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

File: packages/strapi-design-system/webpack.config.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ const entry = fileNames
     return acc;
   }, {});
 
-// Plugin section
+// Plugin <section></section>
```
[the remark this answers is not in the document]
Comment (alexandrebodin) [reply packages/strapi-design-system/webpack.config.js:31]: Nothing :D my prettier went crazy I didn't see it

File: packages/strapi-design-system/webpack.config.js (hunk 1 above)
Comment (alexandrebodin) [reply packages/strapi-design-system/webpack.config.js:61]: If you don't put that webpack won't consider the shared packages as being "big enough" to put them in a common chunk we can play we this to force chunking. 

I'll move it outside of this pr and open a draft one with this

**Review by HichamELBSI — APPROVED** [review]

**Pull request merged**